rfc6962-bis.xml

```diff
@@ -474,7 +474,7 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
       <section title="Log ID" anchor="log_id">
         <figure>
           <preamble>
-            Each log is uniquely identified by an OID (specified in the log's metadata). A log's operator MUST either allocate the OID themselves or request an OID from one of the two Log ID Registries (see <xref target="log_id_registry1"/> and <xref target="log_id_registry2"/>). Various data structures include the DER encoding of this OID, excluding the ASN.1 tag and length bytes, in an opaque vector:
+            Each log is identified by an OID (specified in the log's metadata) which MUST be unique. A log's operator MUST either allocate the OID themselves or request an OID from one of the two Log ID Registries (see <xref target="log_id_registry1"/> and <xref target="log_id_registry2"/>). Various data structures include the DER encoding of this OID, excluding the ASN.1 tag and length bytes, in an opaque vector:
```

Comment:
IIUC, Steve's comment was that he wants the "allocate the OID themselves" option to disappear, because he foresees Log ID collisions if this option is permitted.  I don't share his paranoia.  An organization can only allocate an OID under an OID arc that they own.  If a log operator can't manage their own OID arc properly, then I think they'd probably be just as likely to make the mistake of requesting ONE OID from one of the Log ID Registries and then using that ONE OID for TWO logs.